File: C#/백준/Bronze/5523. 경기 결과/경기 결과.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int cnt1 = 0;
            int cnt2 = 0;
            for (int i=1;i<=a;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                if (aa[0]>aa[1]){
                    cnt1++;
                }
                else if(aa[0]<aa[1]){
                    cnt2++;
                }
            }
            Console.WriteLine($"{cnt1} {cnt2}");
        }
    }
}

File: Java/백준/Bronze/5523. 경기 결과/경기 결과.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int cnt1 = 0;
        int cnt2 = 0;
        for (int i=1;i<=a;i++){
            int n1 = in.nextInt();
            int n2 = in.nextInt();
            if (n1>n2){
                cnt1++;
            }
            else if(n1<n2){
                cnt2++;
            }
        }
        System.out.printf("%d %d",cnt1,cnt2);
    }
}